lib/takeaway.rb

```diff
@@ -0,0 +1,13 @@
+class Takeaway
```

Comment:
Do we need the Takeaway class? It looks like its only responsibility is to delegate to the menu class to display the menu.